Tools/GNUMake/comps/hip.mak

```diff
@@ -19,6 +22,10 @@ else
 endif
 
 #if less than a given version, throw error.
+hipcc_major_lt_4 = $(shell expr $(hipcc_major_version) \< 4)
```

Comment:
I don't think we need to worry about ROCm version right now.  ROCm is under actively development and our target will be the latest version that works for us until it's more stable.